src/metabase/types.clj

```diff
@@ -5,6 +5,7 @@
 (derive :type/Dictionary :type/Collection)
 (derive :type/Array :type/Collection)
 
+(derive :type/Nil :type/*)
```

Comment:
Right now all the `fingerprinter` multimethods (currently 7) would need an additional dispatch alias.